docs/bdew.rst

```diff
@@ -0,0 +1,128 @@
+==================
+BDEW Load Profiles
+==================
+
+Using the demandlib you can create heat and electrical profiles by scaling the BDEW profiles to your desired annual demand.
+The BDEW profiles are the standard load profiles from BDEW.
+
+Heat Profiles
+~~~~~~~~~~~~~
+
+Description
++++++++++++
+
+Heat profiles are created according to the approach described in the `BDEW guideline <https://www.enwg-veroeffentlichungen.de/badtoelz/Netze/Gasnetz/Netzbeschreibung/LF-Abwicklung-von-Standardlastprofilen-Gas-20110630-final.pdf>`_.
+
+The method was originally established in this `PhD Thesis at TU Munich <https://mediatum.ub.tum.de/doc/601557/601557.pdf>`_.
+
+The approach for generating heat demand profiles is described in section 4.1 (Synthetic load profile approach).
+
+.. math::
+
+    Q_{day}(\theta) = KW \cdot h(\theta) \cdot F \cdot SF
+
+| **KW**: Kundenwert (customer value). Daily consumption of customer at :math:`\approx 8 ^\circ C`, depending on SLP type and Temperature timeseries.
+| **h**: h-Wert (h-value) , depending on SLP type and daily mean temperature.
+| **F**: Wochentagsfaktor (week day factor), depending on SLP type and day of the week.
+| **T**: Daily mean temperature 2 meters above the ground (simple mean or "geometric series", which means a weighted sum over the previous days).
+| **SF**: Stundenfaktor (hour factor)
+
+The geometric series approach is meant to account for thermal inertia.
+
+.. math::
+
+   \theta = \frac{T_t + 0.5 \cdot T_{t-1} + 0.25 \cdot T_{t-2} + 0.125 \cdot T_{t-3}}{1 + 0.5 + 0.25 + 0.125}
+
+Depending on the profile type, different coefficients A, B, C, D for the sigmoid function are used.
+
+.. math::
+
+   h(\theta) &= \frac{A}{1+(\frac{B}{\theta-\theta_0})^C} + D \\
+
+   \theta_0 &= 40^\circ C
+
+Types of houses:
```

Comment:
As far as I understand, the profiles include space heating (SH) and domestic hot water (DHW). Thus, they become inaccurate if the insulation is different from the one originally assumed: Demand in summer is dominated by DHW, demand in summer may or may not be dominated by SH - depending on the insulation. (This is also the way, I interpret https://github.com/oemof/demandlib/issues/20.)

Maybe, this limitation should be made explicit.

Reply:
I just revised the general structure and added some information about electrical load profiles. I did not touch the content of the heat profiles.